js/views/modals/purchase/Payment.js

```diff
@@ -99,7 +100,15 @@ export default class extends BaseVw {
         fetchFailed: true,
         fetchError: 'ERROR_INSUFFICIENT_FUNDS',
       });
+      recordEvent('payFromWallet', {
+        currency: getServerCurrency().code,
+        sufficientFunds: false,
+      });
     } else {
+      recordEvent('payFromWallet', {
+        currency: getServerCurrency().code,
+        sufficientFunds: true,
```

Comment:
At this point, it doesn't necessarily mean there are sufficient funds. It just means the client can't confirm that there are. Based on the number of inputs there might not be. The estimate fee api will determine it definitively.